views/layout.php

```diff
@@ -51,35 +48,39 @@
         <body>
             <!-- Mise en place de la topbar -->
             <?php
-                include 'views/'.$topbar.'.php';
+                include ($topbar);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["include" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required. Variables inside are insecure.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)